src/activeprofiles_win.cpp

```diff
@@ -17,15 +18,15 @@
  *
  */
 
-#include "activeprofiles.h"
-#include "applicationinfo.h"
-#include "psicon.h"
-
 #include <QCoreApplication>
-#include <QWidget>
 #include <QTimer>
+#include <QWidget>
```

Comment:
Usually it's a good practice to include 3rdparty headers after application headers. This validates the headers for completeness. You break this rule here.
And be especially careful with `windows.h`. Usually it's order of inclusion may matter a lot. I hope you checked dthe code compiles well.

Reply:
Same logic is take into account for all files.